website/docs/en/guide/advanced/ssr.mdx

```diff
@@ -82,8 +82,8 @@ import { createRsbuild, loadConfig } from '@rsbuild/core';
 
 // Implement SSR rendering function
 const serverRender = (serverAPI) => async (_req, res) => {
-  // Load SSR bundle
-  const indexModule = await serverAPI.environments.ssr.loadBundle('index');
+  // Load Node bundle for SSR
+  const indexModule = await serverAPI.environments.node.loadBundle('index');
```

Comment:
I prefer to use `environments.node` instead of `environments.ssr`. @9aoy what do you think?

Reply:
Prefer `environments.node`. I will check other places where `environments.ssr` is used and correct it